File: csharp/Pokemon/Pokemon/tests/LucarioTest.cs
using NUnit.Framework;
using Pokemon.main;

namespace Pokemon.tests
{
    [TestFixture]
    internal class LucarioTest
    {
        public Lucario Lucario;
        [SetUp]
        public void Setup()
        {
            Lucario = new Lucario();
        }

        [Test]
        public void mainAttack()
        {
            Assert.AreEqual("Attacking opponent with Quick Attack causing damage of 30", Lucario.mainAttack());
        }

        [Test]
        public void secondAttack()
        {
            Assert.AreEqual("Attacking opponent with Quick Attack causing damage of 30", Lucario.secondAttack());
        }
    }
}

File: java/src/test/java/com/iteso/nintendo/LucarioTest.java
package com.iteso.nintendo;

import org.junit.Before;
import org.junit.Test;
import static org.junit.Assert.assertEquals;

/**
 * Created by rvillalobos on 2/24/18.
 */
public class LucarioTest {
    /**
     * Lucario object.
     */
    Lucario lucario;
    /**
     * Set up for the character.
     */
    @Before
    public void setUp(){
        lucario = new Lucario();
    }
    /**
     * Test of the mainAttack;
     */
    @Test
    public void mainAttack(){
        assertEquals("Attacking opponent with Small Fire causing damage of 20", lucario.mainAttack());
    }
    /**
     * Test of the secondAttack;
     */
    @Test
    public void secondAttack(){
        assertEquals("Attacking opponent with Big Fire causing damage of 30", lucario.mainAttack());
    }
}
